nitransforms/patched.py

```diff
@@ -22,7 +22,7 @@ def obliquity(affine):
         The *obliquity* of each axis with respect to the cardinal axes, in radians.
     """
     vs = voxel_sizes(affine)
-    best_cosines = np.abs((affine[:-1, :-1] / vs).max(axis=1))
+    best_cosines = np.abs(affine[:-1, :-1] / vs).max(axis=1)
```

Comment:
The whole obliquity thing has been merged into nibabel. Can you rebase after I update master?